src/connector/src/allow_alter_on_fly_fields.rs

```diff
@@ -94,6 +94,25 @@ mod sink_properties {
 pub static SOURCE_ALLOW_ALTER_ON_FLY_FIELDS: LazyLock<HashMap<String, HashSet<String>>> = LazyLock::new(|| {
     use source_properties::*;
     let mut map = HashMap::new();
+
+      // CDC Properties - added for schema.change.failure.policy
+      map.try_insert(
```

Comment:
nits: extra space at the beginning

Reply:
Talked with @tabVersion , hard code the new added config in `src/connector/src/with_options_test.rs`